frontend/src/types/crawler.ts

```diff
@@ -0,0 +1,110 @@
+export type CrawlState =
```

Comment:
Essentially a copy-paste from `frontend/src/pages/org/types.ts`

Reply:
got it, so just moving to more generic location.